client/web/compose/src/components/PageBlocks/RecordEditor.vue

```diff
@@ -14,12 +14,13 @@
     <div
       v-else-if="module"
       class="mt-3 px-3"
+      :class="recordEditorMode"
     >
       <template v-for="field in fields">
```

Comment:
There is some weird styling if you have horizontal layout also enabled. You can fix this at the end tho.